src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -63,20 +67,43 @@ def _process_model_before_weight_loading(self, model, **kwargs):
         from compressed_tensors.quantization import apply_quantization_config
 
         ct_quantization_config = self.compressor.quantization_config
-        apply_quantization_config(model, ct_quantization_config, run_compressed=True)
 
-    def _process_model_after_weight_loading(self, model, **kwargs) -> None:
-        pass
+        if self.run_compressed and self.is_compressed:
+            apply_quantization_config(model, ct_quantization_config, run_compressed=True)
```

Comment:
note: we still need to be able to run models which are not compressed to start.

Reply:
done